docs/content/message-queuing/nats-streaming.md

```diff
@@ -6,7 +6,7 @@ weight: 64
 
 #### Installation
 
-Fission now install NATS Streaming service bt default while helm installation.
+Fission now installs NATS Streaming service by default while helm installation.
```

Comment:
"Fission installs the NATS streaming service by default when the fission-all helm chart is used for installation"

Reply:
Done

Reply:
Thanks!